app/models/detector.rb

```diff
@@ -1,9 +1,16 @@
 # frozen_string_literal: true
 
-# Detectors are classes that implement various algorithms that allow us to identify patterns
-# within search terms.
-module Detector
-  def self.table_name_prefix
-    'detector_'
-  end
+# == Schema Information
+#
+# Table name: detectors
+#
+#  id         :integer          not null, primary key
+#  name       :string
+#  confidence :float
```

Comment:
I feel waffly about whether we actually were able to state what `confidence` would even mean at the `Detector` level. If we don't know what it's for, I'd like to leave it out until we need it. If I am mistaken and there was a clear plan for what specifically this mean, please remind me...

My recollection is the `detector_category` confidence was what we felt would increment/decrement during the validation process.

Reply:
I don't think we ever finished that conversation. I saw you were thinking about just going with a confidence value on the link record, but Adam hadn't weighed in yet. I'm not sure - it might be needed, but I'm happy to go without it until we realize that for sure. The initial walkthrough seemed to indicate that we might not need a detector confidence.

I'll take it out for now.